lib/streamFromGeojson.js

```diff
@@ -18,10 +19,9 @@ const streamFromGeojson = (featureStream, filePath) => {
   const reader = fs.createReadStream(filePath);
   const parser = geojsonStream.parse();
 
-  reader.on('error', handleError);
+  reader.on('error', (err) => handleError(invalid(err.message)));
```

Comment:
When GeoJSON stream errors, I think that, at this point, means you have invalid JSON — which I think means it should be considered "invalid", eh?